rakelib/utils.rake

```diff
@@ -53,9 +55,9 @@ class Utils
 
   def self.pod_trunk_last_version(pod)
     require 'yaml'
-    stdout, _, _ = Open3.capture3('bundle', 'exec', 'pod', 'trunk', 'info', pod)
+    stdout, = Open3.capture3('bundle', 'exec', 'pod', 'trunk', 'info', pod)
     stdout.sub!("\n#{pod}\n", '')
-    last_version_line = YAML.load(stdout).first['Versions'].last
+    last_version_line = YAML.safe_load(stdout).first['Versions'].last
```

Comment:
Did rubocop comment on this, or did you notice/improve this?
🤯 if rubocop noticed this.

Reply:
Yep, that's rubocop's doing!